### ISSUE TITLE
Feature Request: Color of parent node

### ISSUE THREAD

**Issue by Sydher** [Quorafind/Obsidian-Canvas-MindMap#61]
### Feature Requested

Hi :)  
I would like that when a child node is created, this new node (and its arrow) has the same color as the parent node.
Thanks !

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by rothleer** [comment]
I agree, maybe this plugin could support theme of mindmap.

**Comment by ankit-kapur** [comment]
Should be optional though, I prefer child nodes being the default color.

**Comment by jcesguerram** [comment]
@Quorafind yes please! the option of having the child node automatically be the same color as the parent would be great!
